script.js

```diff
@@ -0,0 +1,115 @@
+/*fetch(url)
+		.then((response) => response.json())
+		.then((json) => console.log(json));
+        fetch_json("http://localhost:3000/Python").then((value) => {
+	console.log(value);
+});
+        
+        
+        */
+const api_url = {
+	python: "http://localhost:3000/Python",
```

Comment:
in the future, you may want to add .env file so you can have different links for each environment (production -developing) read more [here](https://www.freecodecamp.org/news/how-to-use-environment-variables-in-vanillajs/)